### PR TITLE
Updating help for 0.D

### PR DESCRIPTION
**Pull request by Firestorm01X2**
Updating help for 0.D.

I am going to focus only on thigs that is clearly false and misguided.

#### Summary

SUMMARY: Content "Updating help for 0.D"

#### Purpose of change

Dispite being very abstract, now help contain some outdated info. It is better to revise it slightly.

#### Describe the solution

I am going to focus only on things that is clearly false and misguided.

#### Describe alternatives you've considered

Full help rewriting.

#### Additional context

Changed:
* Removed for handguns However, their small size makes them appropriate for short-range combat, **where larger guns fare poorly**. Not really. Greater damage and automatic fire solves the problem. But "Small size " - yes and "appropriate for short-range combat"- yes too.
* Hunting rifles: referenced as Sniper and Marksman rifles. It is effectve on distances "at very long range"  but not 10  tiles as it was long time ago. Also removed mention that they are less accurate upclose.
* Aiming: adding referrence to aiming process.
* CBM: no more "Bionics come in ready-to-install canisters.". It is confusing. Added direct referece to Autodoc, anastetic kits and narcosis.
* Removed FAQ question:
"Q: Shotguns bring in more zombies than they kill!  What's the point?\nA: Shotguns are intended for emergency use. If you are cornered, use your shotgun to escape, then just run from the zombies it attracts.". There is nothing special in shotgun loud shots. Every noncilenced firerarm usually are loud enough to attract a lot of  nearby zombies. Probably it is tip from early day dynamic spawn or something when gun loundess was different. And shotguns "are intended for emergency use" not much more more than other high damage guns. This question definetely overplays shotguns power and role.
* Removed "machineguns are poor melee weapon".
* Removed exsesive warnings about shotgun noise and shogun zombie attraction. Not completely, however. It make sense.
* Removed mentioning of old dynamic spawn.
* Added new FAQ question: "Q: There are too many calibers and guns.  It confuses me - I don't usually understand what is compatible with what.\nA: Try to remember few usual calibers: \n9x19 (or simply 9mm) - fits most basic pistols (Glock, for example) and SMGs, quite easy to find and gets job done versus usual zombies;\n 00 buckshot - for most shotguns. Very powerful against unarmored target at closer ranges;\n .223 (5.56) - for rifles. Good long range option.\n .308 (7.62) - for even more powerful rifles.  Also good long range option.\n  It should be enough in the beginning.  Or even just grab any shotgun, fill it with buckshot and give them hell!".

### CODE REVIEW

File: data/help/texts.json [hunk 1]
```diff
@@ -93,8 +93,9 @@
...
-      "Bionics come in ready-to-install canisters. Installation of a bionic is best left to a trained professional or to specialized medical apparatus. Using machinery to manipulate bionics requires high levels of Intelligence, first aid, mechanics, and electronics. Beware, though, a failure may cripple you!  Many bionic canisters are difficult to find, but may be purchased from certain wandering vagabonds for a very high price.",
-      "As you may note, all of your body parts have only limited space for containing bionics, so you should choose bionics for installation wisely. Of course, any bionic can be removed from your body but it's not any easier than its installation; and as well as installation, this non-trivial surgical procedure requires anesthesia."
+      "Installation of a bionic is only possible with trained professional or to specialized medical apparatus. Using machinery to manipulate bionics requires high levels of Intelligence, first aid, mechanics, and electronics. Beware, though, a failure may cripple you!  Many bionic canisters are difficult to find, but may be purchased from certain wandering vagabonds for a very high price.",
```
Comment (ZhilkinSerg) [inline data/help/texts.json]:
```suggestion
      "Installation of a bionic is only possible with trained professional or to specialized medical apparatus.  Using machinery to manipulate bionics requires high levels of Intelligence, first aid, mechanics, and electronics.  Beware, though, a failure may cripple you!  Many bionic canisters are difficult to find, but may be purchased from certain wandering vagabonds for a very high price.",
```

File: data/help/texts.json [hunk 2]
```diff
@@ -93,8 +93,9 @@
...
+      "Any bionic can be removed from your body but it's not any easier than its installation; and as well as installation, this non-trivial surgical procedure requires anesthesia."
+      "Probably best choice of installing or uninstallings CBM for lone surviros is Autodoc. Usually you can find those in hostipatls and clinincs (even in some 'unnoficial' ones). Any procedure using it requires consuming of single Anesthesia kit. You can bypass this restriction if you find the way to completely ignore pain. But don't even try to bypass this restriction using drugs - it won't help."
```
Comment (ZhilkinSerg) [inline data/help/texts.json]:
```suggestion
      "Probably best choice of installing or uninstallings CBM for lone surviros is Autodoc.  Usually you can find those in hostipatls and clinincs (even in some 'unnoficial' ones).  Any procedure using it requires consuming of single Anesthesia kit.  You can bypass this restriction if you find the way to completely ignore pain.  But don't even try to bypass this restriction using drugs - it won't help."
```

File: data/help/texts.json [hunk 3]
```diff
@@ -148,7 +149,7 @@
...
       "Magazines can be stored inside several worn accessories for quicker access, such as chest rigs and ammo pouches. All these compatible storage items are listed in a given magazine's description. At the moment, you can only store one magazine per clothing item. To store a magazine into a clothing item, 'a'ctivate the appropriate clothing item, at which point you'll get to choose which magazine to store.",
-      "To fire, press <press_fire>, move the cursor to the relevant space, then hit '.' or 'f'. Some guns have alternate firing modes, such as burst fire; to cycle modes, press <press_select_fire_mode>. Firing continuously, especially in bursts, will severely reduce accuracy.",
+      "To fire, press <press_fire>, move the cursor to the relevant space, then use to '.' accurate your aim your gun and 'f' to fire. Additionaly you may want to use fire at predifined aim levels function using 'a', 'c' or 'p'. Some guns have alternate firing modes, such as burst fire; to cycle modes, press <press_select_fire_mode>. Firing continuously, especially in bursts, will severely reduce accuracy.",
```
Comment (ZhilkinSerg) [inline data/help/texts.json]:
```suggestion
      "To fire, press <press_fire>, move the cursor to the relevant space, then use to '.' accurate your aim your gun and 'f' to fire.  Additionaly you may want to use fire at predifined aim levels function using 'a', 'c' or 'p'.  Some guns have alternate firing modes, such as burst fire; to cycle modes, press <press_select_fire_mode>.  Firing continuously, especially in bursts, will severely reduce accuracy.",
```

File: data/help/texts.json [hunk 4]
```diff
@@ -246,7 +247,7 @@
...
-      "<color_brown>( Hunting Rifles</color>\nHunting rifles are popular for their superior range and accuracy. What's more, their scopes or sights make shots fired at targets more than 10 tiles away as accurate as those with a shorter range. However, they are very poor at hitting targets 4 squares or less from the player. Unlike assault rifles, hunting rifles have no automatic fire. They are also slow to reload and fire, so when facing a large group of nearby enemies, they are not the best pick.",
+      "<color_brown>( Sniper and Marksman Rifles</color>\nSniper and marksman rifles are popular for their superior range and accuracy. What's more, their scopes or sights make shots fired at targets more than 30 tiles away as accurate as those with a shorter range. Unlike assault rifles, sniper and marksman rifles ussually have no automatic fire. They are also slow to reload and fire, so when facing a large group of nearby enemies, they are not the best pick.",
```
Comment (ZhilkinSerg) [inline data/help/texts.json]:
```suggestion
      "<color_brown>( Sniper and Marksman Rifles</color>\nSniper and marksman rifles are popular for their superior range and accuracy.  What's more, their scopes or sights make shots fired at targets more than 30 tiles away as accurate as those with a shorter range.  Unlike assault rifles, sniper and marksman rifles ussually have no automatic fire.  They are also slow to reload and fire, so when facing a large group of nearby enemies, they are not the best pick.",
```

**Comment by Firestorm01X2** [comment]
@ZhilkinSerg 
Is double space rule applies here too? Look at other texts
>Though it is relatively rare, certain areas of the world may be contaminated with radiation. It will gr...

There as single spaces

**Comment by Firestorm01X2** [comment]
Jenkins, rebuild.

**Comment by Firestorm01X2** [comment]
>Yeah, double whitespaces should everywhere. This wasn't addressed in help for some unknown reason.

At the same time chagning it everywhere in the help may lead to updating all translations. Probably it is not best idea.

File: data/help/texts.json [hunk 5]
```diff
@@ -148,7 +150,7 @@
...
       "Magazines can be stored inside several worn accessories for quicker access, such as chest rigs and ammo pouches. All these compatible storage items are listed in a given magazine's description. At the moment, you can only store one magazine per clothing item. To store a magazine into a clothing item, 'a'ctivate the appropriate clothing item, at which point you'll get to choose which magazine to store.",
-      "To fire, press <press_fire>, move the cursor to the relevant space, then hit '.' or 'f'. Some guns have alternate firing modes, such as burst fire; to cycle modes, press <press_select_fire_mode>. Firing continuously, especially in bursts, will severely reduce accuracy.",
+      "To fire, press <press_fire>, move the cursor to the relevant space, then use to '.' accurate your aim your gun and 'f' to fire.  Additionaly you may want to use fire at predifined aim levels function using 'a', 'c' or 'p'.  Some guns have alternate firing modes, such as burst fire; to cycle modes, press <press_select_fire_mode>.  Firing continuously, especially in bursts, will severely reduce accuracy.",
```
Comment (Night-Pryanik) [inline data/help/texts.json]:
```suggestion
      "To fire, press <press_fire>, move the cursor to the relevant space, then press '.' to increase aim of your gun and 'f' to fire.  Additionally you may want to fire at predefined aim levels using 'a', 'c' or 'p'.  Some guns have alternate firing modes, such as burst fire; to cycle modes, press <press_select_fire_mode>.  Firing continuously, especially in bursts, will severely reduce accuracy.",
```

File: data/help/texts.json [hunk 6]
```diff
@@ -241,14 +243,14 @@
...
-      "<color_brown>( Hunting Rifles</color>\nHunting rifles are popular for their superior range and accuracy. What's more, their scopes or sights make shots fired at targets more than 10 tiles away as accurate as those with a shorter range. However, they are very poor at hitting targets 4 squares or less from the player. Unlike assault rifles, hunting rifles have no automatic fire. They are also slow to reload and fire, so when facing a large group of nearby enemies, they are not the best pick.",
+      "<color_brown>( Sniper and Marksman Rifles</color>\nSniper and marksman rifles are popular for their superior range and accuracy.  What's more, their scopes or sights make shots fired at targets at very long range tiles away as accurate as those with a shorter range.  Unlike assault rifles, sniper and marksman rifles ussually have no automatic fire.  They are also may be slow to reload and fire, so when facing a large group of nearby enemies, they are not the best pick.",
```
Comment (Night-Pryanik) [inline data/help/texts.json]:
```suggestion
      "<color_brown>( Sniper and Marksman Rifles</color>\nSniper and marksman rifles are popular for their superior range and accuracy.  What's more, their scopes or sights make shots fired at targets at very long range as accurate as those with a shorter range.  Unlike assault rifles, sniper and marksman rifles usually have no automatic fire.  They are also may be slow to reload and fire, so when facing a large group of nearby enemies, they are not the best pick.",
```

File: data/help/texts.json [hunk 7]
```diff
@@ -267,9 +269,9 @@
...
       "Q: Help! I started a fire and now my house is burning down!\nA: Fires will spread to nearby flammable tiles if they are able. Lighting a fire in a set-up brazier, stove, wood stove, stone fireplace, or pit will stop it from spreading. Fire extinguishers can put out fires that get out of control.",
       "Q: I'm cold and can't sleep at night!\nA: Gather some clothes and put them in the place you use to sleep in. Being hungry, thirsty, wet, or injured can also make you feel the cold more, so try to avoid these effects before you go to sleep.",
+      "Q: There is too many calibers and guns. It confuses me - I don't usually understand what is compatible with what.\nA: Try to remember few usual calibers: \n9x19 (or simly 9mm) - fits most basic pistols (glock, for example) and smgs, quite easy to find and gets job done versus usual zombies;\n 00 bucksot - for most shotguns. Very powerfull versus unarmored target at closer ranges;\n  .223 (5.56) - for rifles. Good long range option.\n .308 (7.62) - for even more powerfull rifles.  Also good long range option.\n  It should be enough in the beginning.  Or even just grab any shotgun fill it with buckshot and give them hell!",
```
Comment (Night-Pryanik) [inline data/help/texts.json]:
```suggestion
      "Q: There are too many calibers and guns.  It confuses me - I don't usually understand what is compatible with what.\nA: Try to remember few usual calibers: \n9x19 (or simply 9mm) - fits most basic pistols (Glock, for example) and SMGs, quite easy to find and gets job done versus usual zombies;\n 00 buckshot - for most shotguns. Very powerful against unarmored target at closer ranges;\n .223 (5.56) - for rifles. Good long range option.\n .308 (7.62) - for even more powerful rifles.  Also good long range option.\n  It should be enough in the beginning.  Or even just grab any shotgun fill it with buckshot and give them hell!",
```

**Comment by Firestorm01X2** [comment]
Let's finish for now.
Ready.

**Comment by Firestorm01X2** [comment]
Should I create dublicate PR for master?

File: data/help/texts.json [hunk 8]
```diff
@@ -267,10 +269,10 @@
...
-      "Q: I have a question that's not addressed here. How can I get an answer?\nA: Ask the helpful people on the forum at discourse.cataclysmdda.org or at the IRC channel #CataclysmDDA on freenode."
+      "Q: There are too many calibers and guns.  It confuses me - I don't usually understand what is compatible with what.\nA: Try to remember few usual calibers: \n9x19 (or simply 9mm) - fits most basic pistols (Glock, for example) and SMGs, quite easy to find and gets job done versus usual zombies;\n 00 buckshot - for most shotguns. Very powerful against unarmored target at closer ranges;\n .223 (5.56) - for rifles. Good long range option.\n .308 (7.62) - for even more powerful rifles.  Also good long range option.\n  It should be enough in the beginning.  Or even just grab any shotgun, fill it with buckshot and give them hell!",
+      "Q: I have a question that's not addressed here.  How can I get an answer?\nA: Ask the helpful people on the forum at discourse.cataclysmdda.org, our Discord channel: https://discord.gg/DPxUcX7 or at the IRC channel #CataclysmDDA on freenode."
```
Comment (ZhilkinSerg) [inline data/help/texts.json]: Discord channel is unofficial and should not be mentioned.

**Pull request merged**